include/seastar/http/handlers.hh

```diff
@@ -66,8 +67,7 @@ public:
         return *this;
     }
 
-    std::vector<sstring> _mandatory_param;
-
+    friend class routes;
```

Comment:
This caught my eye. I think it's better to get const getter for _mandatory_param reference, _or_ move the verify_param() helper to be handler_base's method

Reply:
i plan to stuff move stuff into `_mandatory_param`, so we can verify the type of the params right in seastar, instead of in the higher application level. so the const getter is a no-go, as it *exposes* the internals to the public. so i am repurposing this PR to go with the 2nd route.